docs/install/osx_setup.md

```diff
@@ -91,23 +95,29 @@ Install the dependencies, required for MXNet, with the following commands:
 	# For visualization of network graphs
 	pip install graphviz==0.8.4
```

Comment:
If we use pip3, we should change to pip3 all over the doc. Also, as I said this will break for people who uses virtualenv, as I always use pip in my virtualenv. It's up to @aaronmarkham 's call.